.ci/magic-modules/helpers.sh

```diff
@@ -16,5 +16,5 @@ function apply_patches {
   # We keep all the changes (--soft).
   git reset --soft "$(git merge-base HEAD "$4")"
   # Then we commit again.
-  git commit -m "$2" --author="$3" || true  # don't crash if no changes
+  git commit -m "$2" --author="$3" --signoff || true  # don't crash if no changes
```

Comment:
Question - I think this is going to say "signed off by Modular Magician" - is that what you want?  Or do you prefer signed-off-by the actual author?

Reply:
I'm not sure it matters. But wouldn't the original author have to perform the commit to have it signed off by them?

Reply:
Well, git has to believe that the original author did it - but you see here where --author="$3"; we can probably pretend to be that person in this script, if it does matter.